inc/inventorycomputerlib.class.php

```diff
@@ -1093,9 +1093,9 @@ function updateComputer($a_computerinventory, $computers_id, $no_history, $setdy
                      }
 
                      // Encryption details
-                     $input['encryption_tool'] = $disk['encryption_tool'];
-                     $input['encryption_algorithm'] = $disk['encryption_algorithm'];
-                     $input['encryption_type'] = $disk['encrypt_type'];
+                     $input['encryption_tool'] = (isset($disk['encryption_tool']) ? $disk['encryption_tool'] : null);
```

Comment:
```suggestion
                     $input['encryption_tool'] = $disk['encryption_tool'] ??  null;
```

same for the 2 following lines (null coalescing available in php7.0)